### ISSUE TITLE
Cancelling a call immediately after instantiating it

### ISSUE THREAD

**Issue by adekock11** [onsip/SIP.js#413]
Hi guys.

I picked up this behaviour on our web softphone when making a call and then immediately cancelling the call:

```
NS_ERROR_NOT_CONNECTED:  1 sip-0.7.7.js:8312
	send sip-0.7.7.js:8312:7
	[27]</module.exports/InviteClientTransaction.prototype.sendACK sip-0.7.7.js:7778:3
	[27]</module.exports/InviteClientTransaction.prototype.receiveResponse sip-0.7.7.js:7848:9
	onMessage sip-0.7.7.js:8564:15
	connect/this.ws.onmessage sip-0.7.7.js:8422:7
```

Basically, I can see that the code on line 8312 `this.ws.send(message);` is failing, however I would like to know if anybody else has come across this issue and if so, are there any workarounds?

Reproducing this issue:
- Make a new call by using `ua.invite()` method
- With a tiny delay (i.e. almost immediately) call `session.cancel()`

PS: It really is tricky to reproduce, but I've spent some time on this now and it really does occur after a few attempts.

**Comment by egreenmachine** [comment]
I have not seen this happen. Could you include some more logging with siptrace?

**Comment by egreenmachine** [comment]
I believe that this is fixed in 0.11.x versions of SIP.js with the new Transport work. I am going to close this, but if it is continuing to happen please re-open the issue.
